ocs_ci/ocs/vector_utils.py

```
@@ -0,0 +1,448 @@
+            "metadata": {
+                "genre": genres[i % len(genres)],
+                "source_text": f"i am source text {i+1}",
+                "price": (i + 1) * 10,
+            },
+        }
+        vectors.append(vector_obj)
+
+    return vectors
+
+
+def create_vector_bucket(s3_obj, vector_bucket_name, **kwargs):
```

Ruff (0.15.13)

[warning] Missing type annotation for `**kwargs`

(ANN003)